frontend/src/components/static_pages/about.css

```diff
@@ -6,17 +6,23 @@
 
 .images {
   margin: 0;
-  margin-bottom: 40px;
+  margin-bottom: 140px;
 }
 
 .images img {
   width: 400px;
+  height: 400px;
```

Comment:
this doesn't look great at a large desktop size. I think we just want this for screen sizes at 1023px and below? Maybe we should talk with Tina about this requirement

Reply:
On hold until further details on how to treat the photos.

Reply:
Photos have been edited to be 400px x 400px. Please review changes.